.github/workflows/ci-build-binaries.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       uses: ./.github/actions/setup-branch
 
     - name: Install tools
-      run: make install-ci
+      run: make install-build-ci
```

Comment:
what about all other workflows?
```
$ grep -nr 'make install-ci' .github/workflows
.github/workflows/ci-release.yml:62:      run: make install-ci
.github/workflows/ci-unit-tests-go-tip.yml:34:      run: make install-ci
.github/workflows/ci-unit-tests.yml:40:      run: make install-ci
.github/workflows/ci-docker-build.yml:47:      run: make install-ci
.github/workflows/ci-build-binaries.yml:64:      run: make install-ci
.github/workflows/ci-crossdock.yml:44:      run: make install-ci
.github/workflows/ci-elasticsearch.yml:53:      run: make install-ci
.github/workflows/ci-hotrod.yml:43:      run: make install-ci
.github/workflows/ci-all-in-one-build.yml:46:      run: make install-ci
.github/workflows/ci-opensearch.yml:50:      run: make install-ci
```

Reply:
In other workflow file what needs to added build,test can you tell me please

Reply:
only test workflows need the test tools, everything else only build. The CI will fail if we're wrong about that so we'll find out

Reply:
Hello @yurishkuro can you please look the #4859